demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_objects-cardboxes.dplan.scss

```
@@ -38,12 +38,12 @@
 }
 
 .js .card__item:hover {
-    background-color: color.adjust($dp-color-main, $lightness: 35%);
+    background-color: color.adjust($dp-color-brand-main-base, $lightness: 35%);
```

If possible, we do not want to modify color values outside of the design system.

```suggestion
    background-color: $dp-color-brand-main-light-2;
```